taipy/gui/_renderers/factory.py

```diff
@@ -262,7 +262,7 @@ class _Factory:
                 ("on_action", PropertyType.function),
                 ("active", PropertyType.dynamic_boolean, True),
                 ("multiple", PropertyType.boolean, False),
-                ("select_folder", PropertyType.boolean, False),
+                ("selection_type", PropertyType.string, "file"),
```

Comment:
don't set default value here
In this case, it will add the property which is not necessary